### PR TITLE
document s3o for single path

### PR DESCRIPTION
**Pull request by wheresrhys**
@matthew-andrews @AdaRoseEdwards @JakeChampion 

Is there a better way to achieve this though?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,13 @@ var authS3O = require('s3o-middleware');
...
 app.use('/admin', router);
 ```
+If specific paths require auth:
+```js
+var express = require('express');
+var app = express();
+var router = express.Router();
+var authS3O = require('s3o-middleware');
+
+app.get('/', authS3O, router);
```
Comment (matthew-andrews) [inline README.md]: `use`?

Comment (wheresrhys) [inline README.md]: that feels too loose. Then you're advocating allowing any http verb for a url

Comment (matthew-andrews) [inline README.md]: too loose?  wouldn't use just mean all verbs get authed?

Comment (wheresrhys) [inline README.md]: Using `.use()` has the side-effect of e.g. `curl -X DELETE http://site.com/thing` would get handled, whereas before it wouldn't. Or, perhaps more realistic, you might want to s3o PUT, DELETE, or POST, but not GET for a given url: `.use()` won't allow you to do this

Comment (matthew-andrews) [inline README.md]: ok yeah i guess…

**Pull request merged**